backend/routes_cycle_count.py

```
@@ -1,12 +1,33 @@
 from flask import request, jsonify, session
-from models import db, Tool, Checkout, AuditLog, Chemical
+from models import db, Tool, Checkout, AuditLog, Chemical, User
```

Ruff (0.11.9)

`models.User` imported but unused

Remove unused import: `models.User`

(F401)